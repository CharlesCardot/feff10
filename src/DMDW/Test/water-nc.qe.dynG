Dynamical matrix file
                                                                           
  2    3  1 12.0000000  0.0000000  0.0000000  0.0000000  0.0000000  0.0000000
 1  'O   '  14582.560972180485
 2  'H   '  918.7357938396395
    1    1      0.0000000      0.0000000      0.0000000
    2    2      0.1513997     -0.0199584      0.0000002
    3    2     -0.0199584      0.1513997      0.0000002

     Dynamical  Matrix in cartesian axes

     q = (    0.000000000   0.000000000   0.000000000 ) 

  1  1
  1.05436720  0.00000000   -0.22863237  0.00000000    0.00000000  0.00000000
 -0.22863237  0.00000000    1.05436720  0.00000000    0.00000000  0.00000000
  0.00000000  0.00000000    0.00000000  0.00000000    0.00052061  0.00000000
  1  2
 -0.96272323  0.00000000    0.05738542  0.00000000    0.00000000  0.00000000
  0.17139337  0.00000000   -0.09239725  0.00000000    0.00000000  0.00000000
  0.00000000  0.00000000    0.00000000  0.00000000   -0.00005198  0.00000000
  1  3
 -0.09239725  0.00000000    0.17139337  0.00000000    0.00000000  0.00000000
  0.05738542  0.00000000   -0.96272323  0.00000000    0.00000000  0.00000000
  0.00000000  0.00000000    0.00000000  0.00000000   -0.00005198  0.00000000
  2  1
 -0.96261899  0.00000000    0.17107914  0.00000000    0.00000000  0.00000000
  0.05756132  0.00000000   -0.09240971  0.00000000    0.00000000  0.00000000
  0.00000000  0.00000000    0.00000000  0.00000000    0.00192501  0.00000000
  2  2
  0.98237737  0.00000000   -0.15088845  0.00000000    0.00000000  0.00000000
 -0.15091733  0.00000000    0.11151559  0.00000000    0.00000000  0.00000000
  0.00000000  0.00000000    0.00000000  0.00000000    0.00007947  0.00000000
  2  3
 -0.01912713  0.00000000   -0.02006526  0.00000000    0.00000000  0.00000000
  0.09347364  0.00000000   -0.01915908  0.00000000    0.00000000  0.00000000
  0.00000000  0.00000000    0.00000000  0.00000000    0.00042591  0.00000000
  3  1
 -0.09240971  0.00000000    0.05756132  0.00000000    0.00000000  0.00000000
  0.17107914  0.00000000   -0.96261899  0.00000000    0.00000000  0.00000000
  0.00000000  0.00000000    0.00000000  0.00000000    0.00192501  0.00000000
  3  2
 -0.01915908  0.00000000    0.09347364  0.00000000    0.00000000  0.00000000
 -0.02006526  0.00000000   -0.01912713  0.00000000    0.00000000  0.00000000
  0.00000000  0.00000000    0.00000000  0.00000000    0.00042591  0.00000000
  3  3
  0.11151559  0.00000000   -0.15091733  0.00000000    0.00000000  0.00000000
 -0.15088845  0.00000000    0.98237737  0.00000000    0.00000000  0.00000000
  0.00000000  0.00000000    0.00000000  0.00000000    0.00007947  0.00000000

     Dielectric Tensor:

          1.083355865790          0.000753675274          0.000000000000
          0.000753675274          1.083355865790          0.000000000000
          0.000000000000          0.000000000000          1.091316837198

     Effective Charges E-U: Z_{alpha}{s,beta}

     atom #    1
         -0.467233976358          0.087804689140          0.000000000000
          0.087804689140         -0.467233976358          0.000000000000
          0.000000000000          0.000000000000         -0.726759140439
     atom #    2
          0.147390337379         -0.029173488639          0.000000000000
         -0.059093042435          0.296756435872          0.000000000000
          0.000000000000          0.000000000000          0.336367002015
     atom #    3
          0.296756435872         -0.059093042435          0.000000000000
         -0.029173488639          0.147390337379          0.000000000000
          0.000000000000          0.000000000000          0.336367002015

     Diagonalizing the dynamical matrix

     q = (    0.000000000   0.000000000   0.000000000 ) 

 **************************************************************************
     omega( 1) =      -2.020203 [THz] =     -67.387160 [cm-1]
 ( -0.000060  0.000000  0.000111  0.000000  0.000015  0.000000 ) 
 ( -0.000062  0.000000  0.000097  0.000000 -0.707124  0.000000 ) 
 ( -0.000045  0.000000  0.000113  0.000000  0.707090  0.000000 ) 
     omega( 2) =      -1.278560 [THz] =     -42.648456 [cm-1]
 (  0.000411  0.000000 -0.000179  0.000000 -0.437369  0.000000 ) 
 (  0.000392  0.000000 -0.000320  0.000000  0.635798  0.000000 ) 
 (  0.000550  0.000000 -0.000162  0.000000  0.635978  0.000000 ) 
     omega( 3) =      -0.615453 [THz] =     -20.529451 [cm-1]
 ( -0.398622  0.000000  0.390684  0.000000 -0.000673  0.000000 ) 
 ( -0.392594  0.000000  0.435544  0.000000  0.000565  0.000000 ) 
 ( -0.443473  0.000000  0.384664  0.000000 -0.001136  0.000000 ) 
     omega( 4) =       0.375149 [THz] =      12.513713 [cm-1]
 (  0.404427  0.000000  0.412524  0.000000  0.000248  0.000000 ) 
 (  0.404108  0.000000  0.412851  0.000000  0.000332  0.000000 ) 
 (  0.403418  0.000000  0.412027  0.000000 -0.000212  0.000000 ) 
     omega( 5) =       2.823928 [THz] =      94.196736 [cm-1]
 (  0.000168  0.000000 -0.000084  0.000000 -0.128467  0.000000 ) 
 (  0.000170  0.000000 -0.000059  0.000000 -0.701253  0.000000 ) 
 (  0.000142  0.000000 -0.000087  0.000000 -0.701242  0.000000 ) 
     omega( 6) =       2.977630 [THz] =      99.323690 [cm-1]
 (  0.040107  0.000000 -0.040148  0.000000 -0.000011  0.000000 ) 
 (  0.136512  0.000000  0.692619  0.000000  0.000032  0.000000 ) 
 ( -0.692659  0.000000 -0.136552  0.000000  0.000082  0.000000 ) 
     omega( 7) =      46.411982 [THz] =    1548.147369 [cm-1]
 (  0.050309  0.000000  0.050309  0.000000  0.000000  0.000000 ) 
 ( -0.101197  0.000000 -0.698017  0.000000 -0.000001  0.000000 ) 
 ( -0.698017  0.000000 -0.101196  0.000000  0.000001  0.000000 ) 
     omega( 8) =     110.497859 [THz] =    3685.836345 [cm-1]
 ( -0.034748  0.000000 -0.034748  0.000000 -0.000000  0.000000 ) 
 (  0.692216  0.000000 -0.140107  0.000000 -0.000000  0.000000 ) 
 ( -0.140107  0.000000  0.692215  0.000000  0.000000  0.000000 ) 
     omega( 9) =     113.865422 [THz] =    3798.166905 [cm-1]
 (  0.049842  0.000000 -0.049842  0.000000  0.000000  0.000000 ) 
 ( -0.699320  0.000000  0.092016  0.000000  0.000001  0.000000 ) 
 ( -0.092016  0.000000  0.699321  0.000000  0.000001  0.000000 ) 
 **************************************************************************


