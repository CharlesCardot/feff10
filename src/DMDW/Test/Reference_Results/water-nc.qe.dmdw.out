# Lanczos recursion order:    4
# Temperature:   77.00
# Dynamical matrix file: water-nc.qe.dym

--------------------------------------------------------------
 Path Indices:    1   2
 PDOS Poles:
     Freq. (THz)    Weight
        3.159       0.000000061
       46.416       0.002399632
      110.497       0.489728693
      113.865       0.507871615

 PDOS Einstein freq (single pole), associated temp and eff. force constant: 
 Freq (THz)   Temp (K)   Eff. FC (N/m)
 112.113      5380.40     781.3513

 pDOS n Moments, associated Einstein freqs, temps and eff. force constants:
  n     Mom (THz^n)   Freq (THz)     Temp (K)    Eff. FC (N/m)
 -2       0.00008     111.52399      5352.13     773.1640
 -1       0.00894     111.80628      5365.68     777.0831
  0       1.00000     ---------     --------
  1     112.05406     112.05406      5377.57     780.5312
  2   12569.30450     112.11291      5380.40     781.3512

 Path Red. Mass (AMU):    0.948258
 Path Length (Ang), s^2 (1e-3 Ang^2):  0.9697   4.7669
--------------------------------------------------------------
--------------------------------------------------------------
 Path Indices:    1   3
 PDOS Poles:
     Freq. (THz)    Weight
        3.159       0.000000061
       46.416       0.002399632
      110.497       0.489728693
      113.865       0.507871615

 PDOS Einstein freq (single pole), associated temp and eff. force constant: 
 Freq (THz)   Temp (K)   Eff. FC (N/m)
 112.113      5380.40     781.3513

 pDOS n Moments, associated Einstein freqs, temps and eff. force constants:
  n     Mom (THz^n)   Freq (THz)     Temp (K)    Eff. FC (N/m)
 -2       0.00008     111.52399      5352.13     773.1640
 -1       0.00894     111.80628      5365.68     777.0831
  0       1.00000     ---------     --------
  1     112.05406     112.05406      5377.57     780.5312
  2   12569.30450     112.11291      5380.40     781.3512

 Path Red. Mass (AMU):    0.948258
 Path Length (Ang), s^2 (1e-3 Ang^2):  0.9697   4.7669
--------------------------------------------------------------
--------------------------------------------------------------
 Path Indices:    2   1   3
 PDOS Poles:
     Freq. (THz)    Weight
        9.705       0.000000010
       46.410       0.069950394
      110.400       0.000105466
      110.499       0.929943733

 PDOS Einstein freq (single pole), associated temp and eff. force constant: 
 Freq (THz)   Temp (K)   Eff. FC (N/m)
 107.269      5147.94     418.5173

 pDOS n Moments, associated Einstein freqs, temps and eff. force constants:
  n     Mom (THz^n)   Freq (THz)     Temp (K)    Eff. FC (N/m)
 -2       0.00011      95.93794      4604.15     334.7682
 -1       0.00992     100.76561      4835.83     369.3075
  0       1.00000     ---------     --------
  1     106.01625     106.01625      5087.81     408.7976
  2   11506.67386     107.26917      5147.94     418.5171

 Path Red. Mass (AMU):    0.554823
 Path Length (Ang), s^2 (1e-3 Ang^2):  1.9394   9.0398
--------------------------------------------------------------
--------------------------------------------------------------
 Path Indices:    1   2   3
 PDOS Poles:
     Freq. (THz)    Weight
        9.705       0.000000010
       46.410       0.069950394
      110.400       0.000105466
      110.499       0.929943733

 PDOS Einstein freq (single pole), associated temp and eff. force constant: 
 Freq (THz)   Temp (K)   Eff. FC (N/m)
 107.269      5147.94     418.5173

 pDOS n Moments, associated Einstein freqs, temps and eff. force constants:
  n     Mom (THz^n)   Freq (THz)     Temp (K)    Eff. FC (N/m)
 -2       0.00011      95.93794      4604.15     334.7682
 -1       0.00992     100.76561      4835.83     369.3075
  0       1.00000     ---------     --------
  1     106.01625     106.01625      5087.81     408.7976
  2   11506.67386     107.26917      5147.94     418.5171

 Path Red. Mass (AMU):    0.554823
 Path Length (Ang), s^2 (1e-3 Ang^2):  2.5086   9.0398
--------------------------------------------------------------
